### DISCUSSION TITLE
too many heartbeat and acknack

### DISCUSSION THREAD

**Discussion by TechVortexZ** [eProsima/Fast-DDS#5148] [Support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

There are too many heartbeats and acknacks here, how can I reduce it?
Our service discovery uses udp and process communication uses shm.
![image](https://github.com/user-attachments/assets/3c0affbc-b2e1-4e31-a648-6ca41bea2c73)

### Current behavior

too many heartbeats and acknacks

### Steps to reproduce

When I use tcpdump to capture the udp packets during the startup process，There are too many heartbeats。

### Fast DDS version/commit

fastdds-v2.12

### Platform/Architecture

Ubuntu Focal 20.04 amd64

### Transport layer

Default configuration, UDPv4 & SHM

**Comment by TechVortexZ** [comment]
Was my issue so simple or so complicated that no one answered me...

**Comment by JesusPoderoso** [comment]
Hi @TechVortexZ, thanks for using Fast DDS.
Thank you for your patience, we have been focusing our efforts on the upcoming release of Fast DDS v3.0.0 over the past few days.
You can increase the sending period of heartbeats and acknack messages ([WriterTimes](https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/eprosimaExtensions.html#writertimes) configuration) to reduce the traffic. Please refer to [this section](https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/large_data/large_data.html#tuning-heartbeat-period) of the documentation.
I am moving this issue to the Support focum according to the [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md).
